js&css/extension/www.youtube.com/appearance/details/details.css

```diff
@@ -98,61 +98,61 @@ html[it-red-dislike-button='true'] #top-level-buttons-computed #segmented-dislik
 /*--------------------------------------------------------------
 # HIDE SHARE BUTTON
 --------------------------------------------------------------*/
-html[it-hide-share-button='hidden'] #menu button:has(svg path[d^="M15,5.63L20.66,12L15,18.37V15v"]),
-html[it-hide-share-button='hidden'] #flexible-item-buttons button:has(svg path[d^="M15,5.63L20.66,12L15,18.37V15v"]),
+html[it-hide-share-button='hidden'] #menu button:has(svg path[d^="M15 5.63 20.66 12 15 18.37V14h-1c-3.96"]),
+html[it-hide-share-button='hidden'] #flexible-item-buttons button:has(svg path[d^="M15 5.63 20.66 12 15 18.37V14h-1c-3.96"]),
 
-html[it-hide-share-button='icons_only'] #menu ytd-button-renderer:has(svg path[d^="M15,5.63L20.66,12L15,18.37V15v"]) div:nth-child(2),
-html[it-hide-share-button='icons_only'] #menu ytd-button-renderer:has(svg path[d^="M15,5.63L20.66,12L15,18.37V15v"]) .cbox{ 
+html[it-hide-share-button='icons_only'] #menu ytd-button-renderer:has(svg path[d^="M15 5.63 20.66 12 15 18.37V14h-1c-3.96"]) div:nth-child(2),
+html[it-hide-share-button='icons_only'] #menu ytd-button-renderer:has(svg path[d^="M15 5.63 20.66 12 15 18.37V14h-1c-3.96"]) .cbox{ 
 	display: none;
 }
 /*--------------------------------------------------------------
 # HIDE DOWNLOAD BUTTON
 --------------------------------------------------------------*/
-html[it-hide-download-button='hidden'] #menu button:has(svg path[d^="M17 18V19H6V18H17ZM16.5 11.4"]),
-html[it-hide-download-button='hidden'] #flexible-item-buttons button:has(svg path[d^="M17 18V19H6V18H17ZM16.5 11.4"]) {
+html[it-hide-download-button='hidden'] #menu button:has(svg path[d^="M17 18v1H6v-1h11zm-.5-6.6-.7-.7-3.8 3.7V4h-1v10.4l-3.8-3.8-.7.7"]),
+html[it-hide-download-button='hidden'] #flexible-item-buttons button:has(svg path[d^="M17 18v1H6v-1h11zm-.5-6.6-.7-.7-3.8 3.7V4h-1v10.4l-3.8-3.8-.7.7"]) {
```

Comment:
the matches can be shorter predicting minor change. Even a few chars are unlikely to exist twice (and if ever, thats probably better than the feature failing.)